common/utils/convert-image-uri.js

```diff
@@ -100,16 +102,16 @@ export function convertIiifImageUri(
   originalUri: string,
   requiredSize: number | 'full'
 ): string {
-  if (determineIfGif(originalUri)) {
+  if (determineIfGif(originalUri) || originalUri.startsWith(iiifThumbUri)) {
```

Comment:
Actually why not just match `.startWith(iiifImageUri)`

Reply:
good suggestion, done